cluster-provision/k8s/1.18/manifests/cni.diff

```diff
@@ -10,17 +10,6 @@
            "policy": {
                "type": "k8s"
            },
-@@ -3576,8 +3579,8 @@
-             # The default IPv4 pool to create on startup if none exists. Pod IPs will be
-             # chosen from this range. Changing this value after installation will have
-             # no effect. This should fall within `--cluster-cidr`.
--            # - name: CALICO_IPV4POOL_CIDR
--            #   value: "192.168.0.0/16"
-+            - name: CALICO_IPV4POOL_CIDR
-+              value: "10.244.0.0/16"
-             # Disable file logging so `kubectl logs` works.
```

Comment:
I think we need this, do we still get logs?

Reply:
i didnt touch the logs, i touched the 2 above it
and see why reason here please
https://github.com/kubevirt/kubevirtci/pull/422#discussion_r460696582

Reply:
I understand why you drop CIDR variable, my question is why removing lines 22-23
since there is `-` at the start of the lines I assume it will remove them, right?

Do we get logs with `kubectl logs` on the new version?

Reply:
its a diff file with -C 3 (means it shows 3 lines before and after), it actually had zero effect before and also after
it didnt change those line, it just showed the scope

since its a diff file that was changed in non + / - lines , there is zero effect, 
i took just the changes compared to the original and applied only them , removing the unneeded CIDR

the - in the beginning is the diff file of this PR
is a diff of a diff this is why there are 2 columns

if the change of kubectl logs would be touched the diff wont be able to apply